docs/source/markdown/podman-start.1.md

```diff
@@ -38,6 +38,10 @@ to run containers such as CRI-O, the last started container could be from either
 
 Proxy received signals to the process (non-TTY mode only). SIGCHLD, SIGSTOP, and SIGKILL are not proxied. The default is *true* when attaching, *false* otherwise.
 
+#### **--all**, **-a**
```

Comment:
For complicated reasons, you now have to backslash-escape double dashes: `\-\-all`. Look at the lines above for examples.

Reply:
Yeah I saw this in the CI :dango: